index.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  dirname: __dirname,
```

Comment:
Fine with the current, but maybe `specPath = path.join(__dirname, 'spec')` might be more usefull in the long run